frontend/spec/controllers/spree/checkout_controller_spec.rb

```diff
@@ -8,8 +8,8 @@
   let(:order) { FactoryBot.create(:order_with_totals) }
 
   let(:address_params) do
-    address = FactoryBot.build(:address)
-    address.attributes.except("created_at", "updated_at")
+    attributes_for(:address, name: 'John Doe')
```

Comment:
Shouldn't we change the factory?

Reply:
You mean to use ''John Doe" as name or to remove timestamps?
In both cases we will have to update a lot of other specs relying on how address is created now, I don't think it's the right moment to do that